lib/puppet/util/pulp_util.rb

```diff
@@ -50,6 +50,23 @@ def get_repo_info(repo_id)
         request_api("/v2/repositories/#{repo_id}/?details=True")
       end
 
+      def get_role_info(role_id)
+        raise '[get_role_info] Role id should never be nil.' if !role_id
```

Comment:
Convention is to use ```unless``` instead of ```if !```. This also applies to the lines below.

Reply:
Corrected in 31da5bc , thanks